stdout of test 'tst2504` in directory 'monetdb5/mal` itself:


# 08:01:06 >  
# 08:01:06 >  Mtimeout -timeout 60 Mserver "--config=/ufs/mk/monet5/Linux/etc/MonetDB5.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5/Linux/lib/MonetDB5:/ufs/mk/opensource/MonetDB/Linux/lib/MonetDB" --set "gdk_dbfarm=/ufs/mk/monet5/Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/monet5/Linux/var/MonetDB5/log" --set mapi_port=42158 --set sql_port=55708 --set monet_prompt= --trace "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --dbname=mTests_src_mal  tst2504.mal </dev/null
# 08:01:06 >  

# MonetDB server v5.10.0, based on kernel v1.28.0
# Serving database 'mTests_src_mal', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008- MonetDB B.V., all rights reserved
# Visit http://monetdb.cwi.nl/ for further information
# Listening for connection requests on mapi:monetdb://eir.ins.cwi.nl:34870/
!SyntaxException:user.random[0]:RETURN missing
factory user.random(seed:int):int;
    random := seed;
    i := seed;
    i := calc.*(125,i);
    i := calc.%(i,32767);
    yield random := i;
    i := calc.*(125,i);
    i := calc.%(i,32767);
    yield random := i;
end random;
!TypeException:user.main[3]:'user.random' undefined in: r1:int := user.random(_4:int)
!TypeException:user.main[5]:'user.random' undefined in: r2:int := user.random(_4:int)
!TypeException:user.main[7]:'user.random' undefined in: r2:int := user.random(_4:int)
!TypeException:user.main[9]:'user.random' undefined in: r2:int := user.random(_4:int)
function user.main():void;
# The prototypical factory construct 
# test factory return and restart 
    r1 := user.random(1);
    io.print(r1);
    r2 := user.random(1);
    io.print(r2);
    r2 := user.random(1);
    io.print(r2);
    r2 := user.random(1);
    io.print(r2);
end main;

# 08:01:06 >  
# 08:01:06 >  Done.
# 08:01:06 >  

